strictdoc/server/routers/main_router.py

```diff
@@ -2726,6 +2726,54 @@ def get_autocomplete_results(
             status_code=200,
         )
 
+    @router.get("/autocomplete/field", response_class=Response)
+    def get_autocomplete_field_results(
+        q: Optional[str] = None,
+        document_mid: Optional[str] = None,
+        element_type: Optional[str] = None,
+        field_name: Optional[str] = None,
+    ):
+        """
+        Returns matches of possible SingleChoice values of a field
```

Comment:
Let's have this sentence end with a dot. I was looking into enforcing this with Ruff recently but got stuck with some other rules.